playbooks/roles/postgresql_aggregation/defaults/main.yml

```diff
@@ -23,3 +23,12 @@ POSTGRESQL_AGGREGATION_PASSWORD: "password"
 POSTGRESQL_AGGREGATION_USER_ROLE: "boxes"
 POSTGRESQL_AGGREGATION_PORT: 5432
 POSTGRESQL_AGGREGATION_OPTIONS: {}
+
+postgresql_aggregation_client_cert: ''
+postgresql_aggregation_client_key: ''
+postgresql_aggregation_ca_cert: ''
+postgresql_aggregation_client_cert_path: '/etc/ssl/certs/postgresql_aggregation-client-cert.pem'
+postgresql_aggregation_client_key_path: '/etc/ssl/certs/postgresql_aggregation-client-key.pem'
+postgresql_aggregation_ca_cert_path: '/etc/ssl/certs/postgresql_aggregation-server-ca.pem'
+postgresql_aggregation_cert_owner: "{{ edxapp_user }}"
+postgresql_aggregation_cert_group: "{{ edxapp_user }}"
```

Comment:
I agree with @bryanlandia. On most servers, the permissions would be `root:root`.